### PR TITLE
HUI key is no longer hardcoded

### PR DESCRIPTION
**Pull request by ifyio**
cc/ @commuterjoy

### CODE REVIEW

File: sinks/propensity.js [hunk 1]
```diff
@@ -12,7 +12,7 @@ module.exports = function(message) {
 		cookieString         	: message.ingest._headers.cookie,
 		pageUrl				 	: message.egest.annotations.url.href,
 		articlePageUrlPattern	: /next\.ft\.com\/content\/([a-f0-9]{8}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{4}-[a-f0-9]{12})/,
-		propensityUrlTemplate	: 'http://api.ft.com/hui/visitors/{userId}?apiKey=bq67pbt9rpkqm25j978d6e7a'
```
Comment (matthew-andrews) [inline sinks/propensity.js]: is this API key revoked?

Comment (ifyio) [inline sinks/propensity.js]: not sure what you mean

Comment (matthew-andrews) [inline sinks/propensity.js]: do the API gateway team know we leaked the key and have we asked them to revoke it?

Comment (matthew-andrews) [inline sinks/propensity.js]: You can inform them via [#api-tech-support](https://financialtimes.slack.com/messages/api-tech-support/) in Slack ^^

Comment (ifyio) [inline sinks/propensity.js]: I see. ok

**Pull request merged**